internal/integration/stream_test_helpers.go

```diff
@@ -33,7 +33,7 @@ import (
 // specifically requested using the -run flag.
 func CheckSkip(t testing.TB) {
 	if m := flag.Lookup("test.run").Value.String(); m == "" || regexp.MustCompile(strings.Split(m, "/")[0]).FindString(t.Name()) == "" {
-		t.Skip("Skipping as execution was not requested explicitly using go test -run ^TestIntegration$")
+		t.Skip("Skipping as execution was not requested explicitly using go test -run ^Test.*Integration.*$")
```

Comment:
Thought this should reflect whatever's in the Makefile, so it matches all integration test variations.